### DISCUSSION TITLE
Chinese Terminology

### DISCUSSION THREAD

**Discussion by jin-s13** [open-mmlab/mmpose#808] [General]
English | 中文 | 注
-- | -- | --
body keypoint detection | 人体关键点检测
face keypoint detection | 人脸关键点检测
hand keypoint detection | 人手关键点检测
fashion landmark detection | 服饰关键点检测
whole-body keypoint detection | 全身人体关键点检测
body mesh recovery | 人体形状恢复
pose estimation | 姿态检测
pose tracking | 姿态跟踪
3D | 三维
checkpoint | 模型权重文件
backbone | 主干网络
cls_head | 分类头
pipeline | 流水线
hook | 钩子
finetune | 微调
register | 注册器
tensor | 张量
benchmark | 基准测试
detector | 检测器
annotation | 标注
training schedule | 训练策略
scheduler | 调度器
schedule | 调整
ground-truth | ？ | 候选：人工标注
refiner | ? | 候选：精炼器
warm-up | ? | 候选：预热启动
epoch | 训练轮数
iteration | 迭代次数
batchsize | 批处理大小

翻译注意事项： https://github.com/open-mmlab/mmediting/issues/432
